apps/explorer/src/app/routes/blocks/id/index.tsx

```diff
@@ -23,24 +23,29 @@ const Block = () => {
 
   return (
     <section>
-      <h1>BLOCK {block}</h1>
+      <h1 className="route-header">BLOCK {block}</h1>
       <Table>
-        <tr>
-          <td>Mined by</td>
-          <td>
-            <Link to={`/validators/${header.proposer_address}`}>
+        <tr className="table-bordered-tr">
+          <td className="table-bordered-td">Mined by</td>
+          <td className="table-bordered-td">
+            <Link
+              className="text-vega-yellow"
+              to={`/validators/${header.proposer_address}`}
+            >
               {header.proposer_address}
             </Link>
           </td>
```

Comment:
**Nitpick (non-blocking)**: These should be `<th scope=“col”>` elements, as they’re headers. You could also wrap the `<tr>` in a `<thead>` block, but it’s also non-essential. This definitely doesn’t block anything.